tests/h/views/admin/organizations_test.py

```diff
@@ -33,9 +33,8 @@ class TestIndex:
             ("bio", ["BioPub"]),
         ],
     )
-    def test_it_returns_filtered_orgs(
-        self, orgs, pyramid_request, query, expected_orgs
-    ):
+    @pytest.mark.usefixtures("orgs")
```

Comment:
Seeing as this fixture is defined for this one test, it's probably a good candidate for autouse.